src/Cake.Hosts/HostsFileUtil.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Cake.Hosts
+{
+    /// <summary>
+    /// Validate and extract data from hosts file
+    /// </summary>
+    public class HostsFileUtil
```

Comment:
Please make this class and all it's members `internal` as this should not be exposed for external use